icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2725,10 +2722,11 @@ class U_I18N_API FormattedNumber : public UMemory, public FormattedValue {
      * as "foot-and-inch" or "hour-and-minute-and-second".
      *
      * @return `MeasureUnit`.
-     * @draft ICU 68
+     * @stable ICU 68
      */
     MeasureUnit getOutputUnit(UErrorCode& status) const;
 
+#ifndef U_HIDE_DRAFT_API
```

Comment:
Should this one be U_HIDE_INTERNAL_API instead of U_HIDE_DRAFT_API ?

Reply:
Oof.  You're right; I included one method I didn't mean to include.  Fix forthcoming...

Reply:
Sorry; I was looking at the wrong branch.  But yes, after looking at the right branch, I agree-- it looks like this should have been inside U_HIDT_DRAFT_API.  Fix forthcoming...